### PR TITLE
net-im/telegram-desktop: Bump to 3.4.8

### PR DESCRIPTION
**Pull request by mid-kid**
adapted from https://github.com/gentoo/gentoo/pull/23912

@gyakovlev

### CODE REVIEW

**Review by ionenwks — COMMENTED** [review]
This version has the same erroneous QA noise I noted in https://github.com/gentoo/gentoo/pull/23903#pullrequestreview-864946481 and did commit 6b44af01f56461e608e9deb997849728397c5ef3 for (or else the tinderbox will nag and open a bug), although the .desktop is named differently here so adjust accordingly.

(just drive-by mentioning, would rather let gyakolev handle merging the non-bin one).

**Edit:** on a side-note, issue open and few months old upstream https://gitlab.freedesktop.org/xdg/desktop-file-utils/-/issues/59 so I don't think this is getting sorted out soon and adding the QA_ is worth it, I guess could add the link in the ebuild comment for tracking

File: net-im/telegram-desktop/files/tdesktop-3.4.8-jemalloc-only-telegram.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+this is only used during build-time, and jemalloc is merely necessary to
+improve runtime memory use, it's unnecessary to use it for anything else.
+
+--- tdesktop-3.4.8-full.orig/Telegram/CMakeLists.txt
++++ tdesktop-3.4.8-full/Telegram/CMakeLists.txt
+@@ -1331,6 +1331,11 @@ elseif (APPLE)
+         endif()
+     endif()
+ else()
++    target link_libraries(Telegram
```
[the remark this answers is not in the document]
Comment (mid-kid) [reply net-im/telegram-desktop/files/tdesktop-3.4.8-jemalloc-only-telegram.patch:14]: Just goes to show I can't really trust someone else's PR without testing. Sorry about that.

**Comment by mid-kid** [comment]
Fixed concerns.

File: net-im/telegram-desktop/files/tdesktop-3.4.8-jemalloc-only-telegram.patch (hunk 1 above)
[the remark this answers is not in the document]
Comment (gyakovlev) [reply net-im/telegram-desktop/files/tdesktop-3.4.8-jemalloc-only-telegram.patch]: yeah that's the whole reason review process exists =)
sometimes it's not trust/untrust, just people do mistakes or do edits after, without testing again, assuming it'll work.
I build-test 99% prs I merge to catch similar things. but still sometimes let a breakage sip through. good catch, thanks @ionenwks

**Pull request closed without merging**